hivemind/p2p/p2p_daemon_bindings/keys.py

```diff
@@ -0,0 +1,97 @@
+"""
+Originally taken from: https://github.com/mhchia/py-libp2p-daemon-bindings
```

Comment:
Can you please specify that this is taken from [py-libp2p](https://github.com/libp2p/py-libp2p/blob/master/libp2p/crypto/keys.py) (not the p2p bindings themselves)?

This is not only to acknowledge the authors, but to make it clear where the code comes from and why it's written this way.